travis.yml

```diff
@@ -0,0 +1,11 @@
+branches:
+  only:
+    - master
+    - "/^v[0-9.]+$/"
```

Comment:
the version number of the package is still according to the spec, this naming convention is just for the git tags https://git-scm.com/book/en/v2/Git-Basics-Tagging